pkg/chunk/encoding/factory.go

```diff
@@ -63,25 +65,25 @@ type encoding struct {
 
 var encodings = map[Encoding]encoding{
 	DoubleDelta: {
-		Name: "DoubleDelta",
+		Name: "double-delta",
 		New: func() Chunk {
 			return newDoubleDeltaEncodedChunk(d1, d0, true, ChunkLen)
 		},
 	},
 	Varbit: {
-		Name: "Varbit",
+		Name: "varbit",
 		New: func() Chunk {
 			return newVarbitChunk(varbitZeroEncoding)
 		},
 	},
 	Bigchunk: {
-		Name: "Bigchunk",
+		Name: "big-chunk",
```

Comment:
**question:** If anyone is currently using the existing flag will this be a breaking change? Maybe it'd be best to keep the values for encoding types as is.

Reply:
Agreed.